SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WorkerCoordinator/ManagedWorkerCoordinator.cs

```diff
@@ -197,5 +201,20 @@ private static string GetWorkerFlagOrDefault(Connection connection, string flagN
 
             return defaultValue;
         }
+
+        private void WaitForTargetDeploymentReady(Connection connection)
```

Comment:
this should have a timeout. if the deployment ends up in a weird state you might be stuck forever. I think 15 minutes is like the max that it can take? if I remember correctly. it might have changed by now as well.